app/helpers/application_helper.rb

```diff
@@ -1,9 +1,22 @@
 module ApplicationHelper
   def flash_class(level)
     case level
-      when :notice then 'info'
-      when :error then 'error'
-      when :alert then 'warning'
+      when :notice then ''
+      when :error then 'flash-error'
+      when :alert then 'flash-warn'
+    end
+  end
+
+  def sidemenu_link_to(text, path, options = {})
+    ccs_class = "menu-item"
```

Comment:
this variables stores css class that will be assigned to menu item in sidebar. It will be menu-item for every item & 'selected' for those that are links to current path